ckanext/ckanext-apicatalog_routes/ckanext/apicatalog_routes/cli.py

```diff
@@ -65,9 +64,9 @@ def send_harvester_status_emails(ctx, dryrun, force, all_harvesters):
     status_opts = {} if not all_harvesters else {'include_manual': True, 'include_never_run': True}
     status = t.get_action('harvester_status')({}, status_opts)
 
-    errored_runs = any(item.get('errors') != 0 for item in list(status.values()))
-    running = (item.get('started') for item in list(status.values()) if item.get('status') == 'running')
-    stuck_runs = any(_elapsed_since(started).days > 1 for started in running)
+    errored_runs = any(item.get('errors') != 0 for title, item in status.items())
```

Comment:
Is the variable "title" unused in these two lines of code? It seems to me, only "item" is used.